main.py

```diff
@@ -82,6 +102,16 @@ def get_parameters():
     if "MASTER_ENCRYPTION_KEY_ID" in os.environ:
         _args.master_encryption_key_id = os.environ["MASTER_ENCRYPTION_KEY_ID"]
 
+    if "ENVIRONMENT" in os.environ:
+        _args.environment = os.environ["ENVIRONMENT"]
+    else:
+        _args.environment = "NOT_SET"
```

Comment:
you can't have the default here, otherwise it always overrides one passed in on command line
instead set the default in the arg parser
recent changes to others repos (resizer and crown controller) will show this

Reply:
Updated, also formatted with black @dan-hill2802